### PR TITLE
test(k8s): fix placement group test

### PR DESCRIPTION
**Pull request by Sh4d1**
Signed-off-by: Patrik Cyvoct <pcyvoct@scaleway.com>

### CODE REVIEW

File: scaleway/resource_k8s_cluster_beta.go [hunk 1]
```diff
@@ -385,6 +385,10 @@ func resourceScalewayK8SClusterBetaDefaultPoolRead(d *schema.ResourceData, m int
 	defaultPool["updated_at"] = pool.UpdatedAt.String()
 	defaultPool["status"] = pool.Status.String()
 
+	if pool.PlacementGroupID != nil {
+		defaultPool["placement_group_id"] = newRegionalId(region, *pool.PlacementGroupID)
```
Comment (jerome-quere) [inline scaleway/resource_k8s_cluster_beta.go]: placement_group_id should not be regional

File: scaleway/resource_k8s_pool_beta.go [hunk 2]
```diff
@@ -185,6 +185,10 @@ func resourceScalewayK8SPoolBetaRead(d *schema.ResourceData, m interface{}) erro
 	d.Set("created_at", pool.CreatedAt)
 	d.Set("updated_at", pool.UpdatedAt)
 
+	if pool.PlacementGroupID != nil {
+		d.Set("placement_group_id", newRegionalId(region, *pool.PlacementGroupID))
```
Comment (jerome-quere) [inline scaleway/resource_k8s_pool_beta.go]: placement_group_id should not be regional

File: scaleway/resource_k8s_cluster_beta.go (hunk 1 above)
Comment (Sh4d1) [reply scaleway/resource_k8s_cluster_beta.go]: forgot to push the commit :sweat_smile:

File: scaleway/resource_k8s_pool_beta.go (hunk 2 above)
Comment (Sh4d1) [reply scaleway/resource_k8s_pool_beta.go]: same :p

File: scaleway/helpers.go [hunk 3]
```diff
@@ -274,6 +274,13 @@ func newRegionalId(region scw.Region, id string) string {
 	return fmt.Sprintf("%s/%s", region, id)
 }
 
+// newZonedIdFromRegion constructs a unique identifier based on resource region and id
+// but returns a zoned ID with the first zone in the region, i.e. adding `-1` to the region
+// TODO this function is a quick fix
+func newZonedIdFromRegion(region scw.Region, id string) string {
```
Comment (jerome-quere) [inline scaleway/helpers.go]: can you add a comment `deprecated` so tooling cleary mark the function as "should not be used"

Comment (Sh4d1) [reply scaleway/helpers.go]: :heavy_check_mark:

**Pull request merged**